### ISSUE TITLE
Cant download or launch any pack

### ISSUE THREAD

**Issue by rbilicki132** [FTBTeam/FTB-App#1020]
### What Operating System

Windows 11

### Debug Code

FTB-DBGMEHACOGEVA

### Describe the bug

When ever I try to down load a pack it will say "Failed to Install [insert name here] due to an unknown error. Or when I try to launch a pack I get "Failed to prepare instance (name of instance). because.. com.google.gson.jsonSyntaxExceptopn: java.lang.IllegalStateException: Expected BEGIN_Object but string at 1 column 1 pa" then it goes off the screen

### Steps to reproduce

1. open FTB app
2. click play or try to down load any pack

### Expected behaviour

to launch pack or to download pack

### Screenshots

![image](https://github.com/FTBTeam/FTB-App/assets/32137376/754844af-3c17-4a1d-80a8-b5780b0760eb)
![image](https://github.com/FTBTeam/FTB-App/assets/32137376/4c283654-1d54-4170-afb8-c9082b27e1ac)

### Additional information

I have installed overwolf and the app 2 times now with full restarts between each attempt.

**Comment by YTPinkCherry** [comment]
Same

**Comment by MichaelHillcox** [comment]
Could either of you send over your instance.json file for that pack?

You can find this by going into the modpack on the app, then going to settings, open folder, then you should see the instance.json inside the folder that opens

**Comment by MichaelHillcox** [comment]
Also, what do you see when you go to this url? https://launchermeta.mojang.com/mc/game/version_manifest.json

**Comment by rbilicki132** [comment]
For your second question the first line is "{"latest": {"release": "1.20.2", "snapshot": "1.20.3-pre4"}, "versions": [{"id": "1.20.3-pre4", "type": "snapshot", "url": "https://piston-meta.mojang.com/v1/packages/b47ff274bd169d7c834b6fcce55f4950a7f49769/1.20.3-pre4.json"," 
And attached is the instance.json file for a failed install of genesis mod pack.
[instance.json](https://github.com/FTBTeam/FTB-App/files/13513145/instance.json)

I can launch minecraft through the vanilla launcher and if someone sends me the mod pack file I can launch it through other launchers like multiMC

**Comment by MichaelHillcox** [comment]
Thanks for that, can you also attach the following file:

`C:\Users\rbili\AppData\Local\.ftba\bin\versions\version_manifest.json`

**Comment by MichaelHillcox** [comment]
Yup, that would be the problem, can you delete that file and try and launch a pack again?

**Comment by rbilicki132** [comment]
perfect thank you. Can you tell from the logs how it got corrupted/empty?

**Comment by MichaelHillcox** [comment]
Looks like it was just empty. I've forwarded this onto our subsystem developer to ensure fail safes are built into this process.

**Comment by BreadSmoothie** [comment]
Having the same issue. Deleting my version_manifest.json file did not fix it. I am completely lost

**Comment by Gaz492** [comment]
> Having the same issue. Deleting my version_manifest.json file did not fix it. I am completely lost

If you are still having the issue, could you open a seperate issue for us? If deleting the file didn't work for you, you may be runnining into a slightly different issue.
